web/react/components/channel_members.jsx

```diff
@@ -137,6 +137,7 @@ module.exports = React.createClass({
                                     memberList={this.state.member_list}
                                     isAdmin={isAdmin}
                                     handleRemove={this.handleRemove}
+                                    allAdminAccess={true}
```

Comment:
My reasoning behind this was if we ever want to reuse the member_list for something else we might not want everyone to have admin access, so having it there allows it to be specified when the list is originally created whether or not admin access should matter (I originally thought it was being reused by the main team manager already but apparently there's separate jsx files for that).  But if you think that's unnecessary I'm pretty 0/5 on keeping it, it wouldn't be that hard to add in later I 'spose either.